docs/docs/1.0.8/security/secrets.md

```diff
@@ -86,13 +88,81 @@ $ echo "value" | base64
 ```
 :::
 
+### Access to the Secret HTTP API
+
+This API requires authentication. Consult [Accessing Admin Server from a different machine](security/#user-to-control-plane-communication) how to configure remote access.
```

Comment:
link URL needs updating to `/docs/1.0.8/security/certificates/#user-to-control-plane-communication` (yeah, my reorg, sorry about that)